modules/l0/src/main/scala/org/elpaca_metagraph/l0/rewards/ElpacaRewards.scala

```diff
@@ -68,6 +68,29 @@ object ElpacaRewards {
             dataSource.addressesToReward.collect {
               case (address, ds) if ds.epochProgressToReward.value == currentEpochProgress.value => address -> ds.amountToReward
             }
+
+          case dataSource: InflowTransactionsDataSource =>
+            dataSource.existingWallets.foldLeft(Map.empty[Address, Long]) { (acc, dsInfo) =>
+              val (_, ds) = dsInfo
+              ds.addressesToReward.collect {
+                case inflowAddressInfo if inflowAddressInfo.epochProgressToReward.value == currentEpochProgress.value =>
```

Comment:
You should never (almost never) compare by ".value". Classes should be comparable by themselves. Here, you have both classes of the same type - EpochProgress, so you should compare just epoch progress, not the value inside